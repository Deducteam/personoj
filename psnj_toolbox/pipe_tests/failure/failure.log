
{"name":"failure","incr":1,"path":"root","dk":"∀ {prop} (λ p: El prop, p)","tac":null}
