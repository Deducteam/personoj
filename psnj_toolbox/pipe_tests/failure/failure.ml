let () = Testing.run_pipe ~src:"failure.log" Sys.argv.(1)
