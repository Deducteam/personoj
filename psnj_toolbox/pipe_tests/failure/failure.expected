=> failure.lp:
require open pipe.encoding.lhol pipe.encoding.propositional_connectives;
require pipe.spec;
symbol failure!1 : @Prf (all prop (λ (p : El prop), p)) ≔
begin
why3;
end;
=> check failed: [3:7-16] "Alt-Ergo" did not find a proof
